src/python/WMCore/MicroService/MSTransferor/MSTransferor.py

```diff
@@ -33,6 +35,14 @@
 from WMCore.Services.Rucio.RucioUtils import GROUPING_ALL
 
 
+def errorString(reason, data, result, status=None)):
```

Comment:
I don't see it used and still don't have the context, so just leaving this message here.

Reply:
ok, removed